File: backjoon/10814.cpp
#include<iostream>
#include<string.h>
using namespace std;

class ssort{
public:
    int num;
    int age;
    string name;
}Ssort;

int main(){
    int N;
    ssort arr[N];
    for(int i=0;i<N;i++){
        arr[i].num=i;
        cin>>arr[i].age>>arr[i].name;
    }
    for(int i=0;i<N;i++)
        cout<<arr[i].age<<" "<<arr[i].name<<endl;
}

File: backjoon/10814.py
import sys
input = sys.stdin.readline

n=int(input())
arr=[]

for i in range(n):
    age,name = input().split()
    age=int(age)
    arr.append((age,name))
arr.sort(key=lambda x : x[0])

for age,name in arr:
    print(age,name)